src/ds/ccf_assert.h

```diff
@@ -56,7 +56,7 @@
 #  define PBFT_FAIL(msg) \
     do \
     { \
-      LOG_INFO << " FATAL_ERROR: " << (msg) << "\n"; \
+      LOG_INFO_FMT(" FATAL_ERROR: {}",(msg)); \
```

Comment:
Nit: Should these be at `FAIL` or `FATAL` level, rather than `INFO`?

Reply:
We don't then need the FATAL_ERROR bit.